src/openrct2/actions/SetCheatAction.hpp

```diff
@@ -761,17 +759,16 @@ DEFINE_GAME_ACTION(SetCheatAction, GAME_COMMAND_CHEAT, GameActionResult)
         // Completely unown peep spawn points
         for (const auto& spawn : gPeepSpawns)
         {
-            int32_t x = spawn.x;
-            int32_t y = spawn.y;
-            if (x != PEEP_SPAWN_UNDEFINED)
+            auto peepSpawn = CoordsXY{ spawn.x, spawn.y };
+            if (peepSpawn.x != PEEP_SPAWN_UNDEFINED)
```

Comment:
This whole `if` can be removed as this is no longer required as `PEEP_SPAWN_UNDEFINED` is not possible.

Reply:
I removed it, but why is it not possible anymore?

Reply:
Because it's now a vector rather than a fixed-length array. Any removed peep spawns are removed from the vector.

Reply:
Ah, ok. Thanks 👍